### PR TITLE
Add substitution for delivry date

### PR DESCRIPTION
**Pull request by BB2A-Anthony**
# New [*Add substitution for delivry date*]

### CODE REVIEW

File: htdocs/core/lib/functions.lib.php [hunk 1]
```diff
@@ -6380,8 +6380,14 @@ function getCommonSubstitutionArray($outputlangs, $onlykey = 0, $exclude = null,
 			$substitutionarray['__REF_SUPPLIER__'] = (isset($object->ref_supplier) ? $object->ref_supplier : null);
 			$substitutionarray['__NOTE_PUBLIC__'] = (isset($object->note_public) ? $object->note_public : null);
 			$substitutionarray['__NOTE_PRIVATE__'] = (isset($object->note_private) ? $object->note_private : null);
-
 			$substitutionarray['__DATE_DELIVERY__'] = (isset($object->date_livraison) ? dol_print_date($object->date_livraison, 'day', 0, $outputlangs) : '');
+			$datedelivery = dol_getdate($object->date_livraison, true);
+			$substitutionarray['__DATE_DELIVERY_DAY__'] = (isset($object->date_livraison) ? sprintf("%02d", $datedelivery['mday']) : '');
```
Comment (eldy) [inline htdocs/core/lib/functions.lib.php:6385]: To get the day, hour, month, etc of a date, use instead   
dol_print_date("%d", $object->date_livraison)
dol_print_date("%H", $object->date_livraison)
...

Comment (bb2a) [reply htdocs/core/lib/functions.lib.php]: @eldy I have update

Comment (eldy) [reply htdocs/core/lib/functions.lib.php]: We've got error in CI due to syntax of code. No space found after comma in argument list

Comment (bb2a) [reply htdocs/core/lib/functions.lib.php]: @eldy I have update

**Pull request merged**